### DISCUSSION TITLE
How do I change the item despawning time for a specific item from the default 5 minutes in Fabric?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2796] [Mod Dev Support]
How do I change the item despawning time for a specific item from the default 5 minutes in Fabric?

**Comment by warjort** [comment]
The limit is hardwired, see the bottom of the tick method.

But you can change the behaviour with a mixin.
e.g. this is Botania messing with the age for its items:
https://github.com/VazkiiMods/Botania/blob/1.19.x/Xplat/src/main/java/vazkii/botania/mixin/ItemEntityMixin.java
